### ISSUE TITLE
Sending with BTCPay

### ISSUE THREAD

**Issue by delmol** [btcpayserver/btcpayserver#73]
Not sure whether this is currently possible with BTCPay, but can you also send payments out?

BTCPay offers a great way to programmatically generate invoices and receive payments for them.

What I'm trying to do is also send payments out programmatically for refunds, withdrawals etc.

Does BTCPay offer a way to do this, or am I best just interfacing directly with the bitcoin/litecoin daemon to send payments?

**Comment by NicolasDorier** [comment]
Yes, but only if you are using a Ledger Nano S. There is an integrated wallet (go to store settings / Derivation Scheme / Wallet)

It is planned for lightning as well.

**Comment by delmol** [comment]
Would that require the ledger to authorise each transaction, or can it
automatically send the payments?

I'm trying to send them automatically, so some sort of API call or similar
would be best:

Send ($amount, $address);

I'm not sure if this is possible though.

I suppose an alternative could be to send the money from BTCpay to another
full node which handles outgoing payments

**Comment by NicolasDorier** [comment]
@delmol it require the Ledger Nano S yes.

Your alternative is to host your keys on Electrum, increase the gap limit, then use Electrum's key.

**Comment by jonathan-daniel** [comment]
Is this still not supported with the new API?

**Comment by ilearnio** [comment]
@NicolasDorier According to this article https://docs.btcpayserver.org/LedgerWallet/ the Ledger Nano S is no longer supported. Is there any alternative for the withdrawals? We have a deposit feature on our app and want to give users a way to withdraw their funds.

**Comment by Kukks** [comment]
> @NicolasDorier According to this article https://docs.btcpayserver.org/LedgerWallet/ the Ledger Nano S is no longer supported. Is there any alternative for the withdrawals? We have a deposit feature on our app and want to give users a way to withdraw their funds (need to allow them to enter any amount and a wallet address).

It is supported through a newer approach https://docs.btcpayserver.org/HardwareWalletIntegration/

**Comment by ilearnio** [comment]
So there’s still no such option for withdrawals in btcpay, right? So two alternatives would be:

- Use Electrum’s API
- Use Ledger Nano hardware wallet
